rpcs3/util/fixed_typemap.hpp

```diff
@@ -205,11 +240,36 @@ namespace stx
 			return init<T>(std::forward<Args>(args)...);
 		}
 
-		// Obtain object pointer (thread safe just against other get calls)
+		template <typename T>
+		bool is_init() const noexcept
+		{
+			return m_init[stx::typeindex<typeinfo, std::decay_t<T>>()];
+		}
+
+		// Obtain object pointer (may be uninitialized memory)
 		template <typename T>
 		T* get() const noexcept
 		{
-			return static_cast<T*>(m_list[stx::typeindex<typeinfo, std::decay_t<T>>()]);
+			if constexpr (Size != 0)
+			{
+				return std::launder(reinterpret_cast<T*>(m_data + stx::typeoffset<typeinfo, std::decay_t<T>>()));
+			}
+			else
+			{
+				return std::launder(reinterpret_cast<T*>(m_list + stx::typeoffset<typeinfo, std::decay_t<T>>()));
+			}
+		}
+
+		// Obtain object pointer if initialized
+		template <typename T>
+		T* try_get() const noexcept
```

Comment:
Maybe can be written after pr but I think get() should return a reference and not a pointer to emphasize the fact that it won't return nullptr on failure.

Reply:
I technically agree but too many spam changes.